lib/watir-webdriver/browser.rb

```diff
@@ -10,6 +10,7 @@ class Browser
     include Waitable
 
     attr_reader :driver
+    attr_reader :checkers
```

Comment:
I actually like "Checkers" or maybe I just got used to them. Maybe `AfterHooks` is a better name, but I don't really have a solid opinion on this.